content/en/blog/_posts/2023-03-08-Updates-to-the-Auto-refreshing-Official-CVE-Feed/index.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blog 
+title: Updates to the Auto-refreshing Official CVE Feed
+date: 2023-02-08
+slug: k8s-cve-feed-beta
+---
+
+**Author**: Cailyn Edwards (Shopify)
+
+Since launching the [Auto-refreshing Official CVE feed](/docs/reference/issues-security/official-cve-feed/) as an `alpha`
+feature in the 1.25 release we have made signficant improvments and updates. We are excited to announce the release of the 
+`beta` version of the feed. This blog post will outline the changes made, and talk about what is planned for the to expect for 
+the `stable` release.
+
+## Updates
+| **\#** | **Title**                                                                                                    | **Issue**                                                       | **Status**                                                                                                                                                                                                                      |
+| ------ | ------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1 | Support RSS feeds by generating data in Atom format | [kubernetes/sig-security#77](https://github.com/kubernetes/sig-security/issues/77) | open, addressed by [kubernetes/website#39513](https://github.com/kubernetes/website/pull/39513)|
+| 2 | CVE Feed: Sort Markdown Table from most recent to least recently announced CVE | [kubernetes/sig-security#73](https://github.com/kubernetes/sig-security/issues/73) | open, no PR open |
```

Comment:
This PR fixes it: https://github.com/kubernetes/sig-security/pull/76

Reply:
Did it? I did not realize 😅